docs/admin/users/idp-sync.md

```diff
@@ -304,77 +339,141 @@ Visit the Coder UI to confirm these changes:
-Set the following in your Coder server [configuration](../setup/index.md).
+   ```console
+   coder organization settings show org-sync
+   {
+      "field": "organizations",
+      "mapping": {
+         "product": ["868e9b76-dc6e-46ab-be74-a891e9bd784b", "cbdcf774-9412-4118-8cd9-b3f502c84dfb"]
+      },
+      "organization_assign_default": true
+   }
+   ```
 
-```env
-CODER_OIDC_ORGANIZATION_MAPPING='{"data-scientists":["d8d9daef-e273-49ff-a832-11fe2b2d4ab1", "70be0908-61b5-4fb5-aba4-4dfb3a6c5787"]}'
-```
+1. Update with the JSON payload. In this example, `settings.json` contains the
+   payload:
 
-> One claim value from your identity provider can be mapped to many
-> organizations in Coder (e.g. the example above maps to 2 organizations in
-> Coder.)
+   ```console
+   coder organization settings set org-sync < settings.json
+   {
+      "field": "organizations",
+      "mapping": {
+         "product": [
+            "868e5b23-dc6e-46ab-be74-a891e9bd784b",
+            "cbdcf774-4123-4118-8cd9-b3f502c84dfb"
+         ],
+         "sales": [
+            "d79144d9-b30a-555a-9af8-7dac83b2q4ec",
+         ]
+      },
+      "organization_assign_default": true
+   }
+   ```
 
-By default, all users are assigned to the default (first) organization. You can
-disable that with:
+   Analyzing the JSON payload:
 
-```env
-CODER_OIDC_ORGANIZATION_ASSIGN_DEFAULT=false
-```
+   | Field                       | Explanation                                                                                                                                                                                                                                                                             |
+   | :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+   | field                       | If this field is the empty string `""`, then org-sync is disabled. </br> Org memberships must be manually configured through the UI or API.                                                                                                                                             |
+   | mapping                     | Mapping takes a claim from the IdP, and associates it with 1 or more organizations by UUID. </br> No validation is done, so you can put UUID's of orgs that do not exist (a noop). The UI picker will allow selecting orgs from a drop down, and convert it to a UUID for you.          |
+   | organization_assign_default | This setting exists for maintaining backwards compatibility with single org deployments, either through their upgrade, or in perpetuity. </br> If this is set to 'true', all users will always be assigned to the default organization regardless of the mappings and their IdP claims. |
+
+### Dashboard
```

Comment:
```suggestion
### Enviroment Config
```

This reads `dashboard`, but these docs are configuring via env vars. We should change this title, and have some warning like **NOT RECOMMENDED** and **will be overwritten by database settings**.

We do need a `Dashboard` section, and that section should be the default. @jaaydenh just recently finished the UI.

Reply:
@Emyrk I merged in the Dashboard PR, so these steps are now updated - that includes removing the env vars steps

Reply:
done 

![2024-12-10_10-48-29](https://github.com/user-attachments/assets/480ffc94-d435-4439-bac4-84b77733f722)